application/event_processor/event_processor.py

```diff
@@ -29,6 +29,7 @@
 tracer = Tracer()
 lambda_client = boto3.client("lambda")
 expected_env_vars = ("DB_SERVER", "DB_PORT", "DB_NAME", "DB_SCHEMA", "DB_USER_NAME", "EVENT_SENDER_LAMBDA_NAME")
+UN_MATCHED_PHARMACY_LOG_ID = "UN_MATCHED_PHARMACY"
```

Comment:
Can we move all the reporting/logging functionality into a new/another file called something like `reporting.py`. We will eventually have many reporting/logging functions so it isn't feasible to keep them all in this `event_processor.py` file. I've done the same in DI-99 and am happy to rebase off whatever you do if this PR does in first.

Reply:
I was thinking the same. completely agree with you. I will wait for you to merge first and then rebase, which is easy for me.